mbedtls/Cargo.toml

```diff
@@ -23,23 +23,23 @@ keywords = ["MbedTLS", "mbed", "TLS", "SSL", "cryptography"]
 features = ["x509", "ssl"]
 
 [dependencies]
-bitflags = "1"
-serde = { version = "1.0.7", default-features = false, features = ["alloc"] }
-serde_derive = "1.0.7"
+bitflags = "2.6"
+serde = { version = "1.0.203", default-features = false, features = ["alloc"] }
+serde_derive = "1.0.203"
 byteorder = { version = "1.0.0", default-features = false }
-yasna = { version = "0.2", optional = true, features = [
+yasna = { version = "0.5", optional = true, features = [
     "num-bigint",
     "bit-vec",
 ] }
-num-bigint = { version = "0.2", optional = true }
-bit-vec = { version = "0.5", optional = true }
+num-bigint = { version = "0.4", optional = true }
+bit-vec = { version = "0.6", optional = true }
 cbc = { version = "0.1.2", optional = true }
 rc2 = { version = "0.8.1", optional = true }
 cfg-if = "1.0.0"
 tokio = { version = "1.16.1", optional = true }
-chrono = { version = "0.4", optional = true }
+chrono = { version = "0.4.38", optional = true }
 
-mbedtls-sys-auto = { path = "../mbedtls-sys", version = "2.25.0", default-features = false, features = [
+mbedtls-sys-auto = { path = "../mbedtls-sys", version = "2", default-features = false, features = [
```

Comment:
Why the downgrade?

Reply:
mbedtls-sys latest version is '2.28.7' so keeping it '2' here should fetch latest one.

Reply:
You should use `2.28` instead of `2`

Reply:
Please revert to "2.25.0" or change to higher.
reduce version requirement is not allowed in semver